MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m

```diff
@@ -106,7 +106,7 @@ static NSInteger bit_binaryImageSort(id binary1, id binary2, void *__unused cont
   for (unsigned int i = 0; i < images_count; ++i) {
```

Comment:
In this file there are many comments  in code blocks using
```
/**
 * Multiline 
 * comment
 */
```

Where it should instead be
```
/*
 * Multiline 
 * comment
 */
```